### DISCUSSION TITLE
grpc-swift integration

### DISCUSSION THREAD

**Discussion by mateuszfidos** [vyshane/grpc-swift-combine#22] [General]
Hey! I'm trying to build a framework that's built on `grpc-swift` among some other ones but I'm having problems `linting` my `.podspec.` I keep getting linker errors related to `NIOPosix` and some `symbol(s) not found for architecture arm64`. I thought that maybe I'm doing smth wrong so I decided to check out your library - as it also depends on `grpc-swift` - but when I'm `linting` your podspec locally (`pod lib lint`) I get the same errors. 

Have you experienced any issues while `linting`? Maybe you know about this issue and have a solution? 
Any help would be appreciated.

I'm on mac m1, osx 12.1, xcode 13.2.1, cocoapods v.1.11.2

**Comment by vyshane** [comment]
I'm not near a computer now but one thing comes to mind. Have you tried doing it from an Intel Mac? I've had problems linting from m1 before as well, while it worked from Intel.

I would recommend switching to SPM if you can. That seems to be the primary supported environment for Swift gRPC. Their podspec lags behind their SPM package for most releases, and it's also the environment that I personally dogfood for my library.

**Reply by mateuszfidos** [reply]
No I haven't but I'll definitely try that out - lucky enough I still have my old intel mac around. Yeah, I guess I do have to switch to SPM, hopefully those other dependencies also support it. Thanks @vyshane, really appreciate the help.
